.travis.yml

```diff
@@ -86,6 +86,8 @@ env:
   global:
   # Disable Ryuk resource reaper in travis jobs since we always spin up fresh VMs
   - TESTCONTAINERS_RYUK_DISABLED=true
+  # Override default akka.actor.typed.testkit.timefactor
+  - AKKA_TEST_TIMEFACTOR=5.0
```

Comment:
I understand that you are trying a few things here. Do you need any input or fresh ideas?

In general the default factor of 1.0 should be enough, also in Travis. If we have specific things that take longer time to initialize, such as Testcontainers I think we should have long explicit timeouts for those things.

Reply:
Thanks for the comment. The integration test has assertions that fail for a couple reasons: 1) the projection doesn't exist yet because no events have flowed, and 2) the assertion failed because some of the events have flowed, but not all have been processed yet. As a first step I'm trying to inflate the time it takes for a single expectation to rule out any possible bugs. If that works then we could leave it at that, or I could further investigate why it's slow. The testcontainer will already initialize the container to a ready state with an internal health check (AFAICT), but in some cases (maybe 1/100?) it's not long enough.

Reply:
Some ideas for a custom timeout instead of setting it for the whole project:

1. Override the timefactor for `KafkaToSlickIntegrationSpec` only
2. Add a sleep to `beforeAll` `KafkaToSlickIntegrationSpec`
3. Use an override of `ProjectionTestKit.run` that takes a custom duration

Reply:
1. I don't think timefactor is the way to go, will just hide the problems.
2. maybe not sleep, but there should be a way to verify that the Kafka testcontainer is up and running before proceeding with the real test.
3. there is already `def run(projection: Projection[_], max: FiniteDuration)` but if we know that the environment is up before we start the real test that might not be needed.
4. yes, something like that if it's not possible to verify with the testcontainer api

Reply:
I added an additional startup check to `KafkaContainerCluster` upstream in Alpakka Kafka that produces to and consumes from a topic.

https://github.com/akka/alpakka-kafka/pull/1131

I'm going to test it locally with a snapshot, but the real test will be on travis.  Once that PR is merged I'll use the snapshot and continue testing, without the custom timeouts.